_layouts/obs.html

```diff
@@ -23,7 +23,7 @@ <h1 itemprop="headline">
                         <button id="download_menu_button" class="btn btn-secondary dropdown-toggle" type="button" data-toggle="dropdown"><span class="glyphicon glyphicon-cloud-download"></span> Download
                             <span class="caret"></span></button>
                         <ul class="dropdown-menu">
-                            <li><a type="submit" onclick="window.open(getDownloadUrl())"><span class="glyphicon glyphicon-compressed"> MD</span></a></li>
+                            <li><a type="submit" onclick="window.open(getDownloadUrl())"><span id="download_menu_source_item" class="glyphicon glyphicon-compressed"> Markdown</span></a></li>
```

Comment:
This file is no longer used per comment in the header. We should actually delete it now that nothing calls it.